docs/DeveloperGuide.md

```diff
@@ -195,6 +210,36 @@ The following sequence diagram shows how a list is autosorted upon an addition o
     * Pros: Relatively low time complexity i.e. O(logn).
     * Cons: Prone to error and difficult to implement.
 
+### Done feature
+
+#### Implementation
+
+This Done feature allows users to mark their deliverables as completed. The user input is received by`MainWindow` in
```

Comment:
I think 'The Done feature' instead of 'This Done feature' would be more appropriate